ferretdb_packaging/defineversion/main.go

```diff
@@ -0,0 +1,278 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main defines debian version number of DocumentDB for CI builds.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"slices"
+	"strings"
+
+	"github.com/sethvargo/go-githubactions"
+)
+
+func main() {
+	controlFileF := flag.String("control-file", "../pg_documentdb/documentdb.control", "pg_documentdb/documentdb.control file path")
+
+	// https://docs.github.com/en/rest/git/refs?apiVersion=2022-11-28#list-matching-references
+	tagsF := flag.String("tags", "", "release tags of microsoft/documentdb in JSON format")
```

Comment:
This might not be totally necessary, it is used to check that existing released version of microsoft/documentdb from result of github action like below.

```
      - name: Get latest release
        uses: octokit/request-action@v2.x
        id: tags
        with:
          route: GET /repos/microsoft/documentdb/git/matching-refs/tags
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      - run: "echo latest release: '${{ steps.tags.outputs.data }}'"
```